lib/boom.ex

```diff
@@ -42,9 +43,21 @@ defmodule Boom do
           ErrorStorage.clear_errors(notification_trigger, error_kind)
         end
       rescue
-        # FIXME: we should handle this in a different way
-        # credo:disable-for-next-line
-        e -> IO.inspect(e, label: "[Boom] Error sending exception")
+        e ->
+          error_info = Exception.format_banner(:error, e, __STACKTRACE__)
+
+          [first_stack_entry | _] = __STACKTRACE__
+
+          # Will transform '(boom) test/notifier_test.exs:32: NotifierTest.FailingNotifier.notify/2'
+          # into 'NotifierTest.FailingNotifier.notify/2'
+          [_failing_notifier_file, failing_notifier] =
+            first_stack_entry
+            |> Exception.format_stacktrace_entry()
+            |> String.split(~r{:\d+:\s})
```

Comment:
Could this be a match error in case `String.split` doesn't return a List with at least 2 elements?

Reply:
Yes, I agree we could eventually have a match error here so I'm gonna give this a second thought. Do you have any suggestion?

Reply:
Maybe a `case` and have a default clause to match anything and just print that

Reply:
I think this might also work and I find it less obscure than my first solution. Let me know if you find any pitfall 

```elixir
          failing_notifier =
            first_stack_entry
            |> Exception.format_stacktrace_entry()
            |> String.split()
            |> List.last()
```

It will just print the whole formated stacktrace entry if it cannot split it.

Reply:
Nevermind, I think I've found a better solution